converting-audio-to-text-and-translation-using-openai/main.bal

```diff
@@ -0,0 +1,50 @@
+import ballerina/io;
+import ballerinax/openai;
+
+// OpenAI token
+configurable string openAIToken = ?;
+
+const string audioFilePath = "./audo_clips/german-hello.mp3";
+const string audioFile = "english-hello.mp3";
+const string translatingLanguage = "French";
+
+// OpenAI API client configuration
+openai:OpenAIClient openaiClient = check new ({
```

Comment:
Does this have to be at module-level? Can't it be a local variable instead (since it's a main and not a service).